lib/atlas/scaler.rb

```diff
@@ -0,0 +1,68 @@
+module Atlas
+  class Scaler
+    UNSCALED_ETENGINE_DATA_FILES = %w( fce load_profiles network ).freeze
+
+    def initialize(base_dataset_key, derived_dataset_name, number_of_residences, base_value = nil)
```

Comment:
But by the scenario migration rake task in etengine that i am going to push, hopefully tomorrow